code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -576,13 +576,17 @@
 	if(usr.incapacitated())
 		return
 
-	if(ishuman(usr) || isMoMMI(usr))
-		if(isMoMMI(usr))
-			src.add_hiddenprint(usr)
-			add_fingerprint(usr)
+	if (isrobot(usr))
```

Comment:
This isn't your fault (I assume you copypasted this bit from somewhere else), but do not use `usr`. Use `user` instead.

Reply:
I was taught about that but it's being used in a verb not a proc?

Reply:
it's okay in a verb, I think shifty was confused by the way the diff presents this